File: MPP_Lab_Proiect/src/domain/Locatie.java
package domain;

public class Locatie {
    private String tara;
    private String oras;

    public Locatie(String tara, String oras) {
        this.tara = tara;
        this.oras = oras;
    }

    public String getTara() {
        return tara;
    }

    public void setTara(String tara) {
        this.tara = tara;
    }

    public String getOras() {
        return oras;
    }

    public void setOras(String oras) {
        this.oras = oras;
    }

    @Override
    public String toString() {
        return "Locatie{" +
                "tara='" + tara + '\'' +
                ", oras='" + oras + '\'' +
                '}';
    }
}

File: MPP_Proiect/MPP_Proiect/domain/Locatie.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MPP_Proiect.domain
{
    class Locatie
    {
        private string tara;
        private string oras;

        public Locatie(string tara, string oras)
        {
            this.tara = tara;
            this.oras = oras;
        }

        public string getTara()
        {
            return tara;
        }

        public void setTara(string tara)
        {
            this.tara = tara;
        }

        public string getOras()
        {
            return oras;
        }

        public void setOras(string oras)
        {
            this.oras = oras;
        }

        public override string ToString()
        {
            return "Locatie{" +
                "Tara='" + tara + '\'' +
                ", Oras='" + oras + '\'' +
                '}';
        }
    }
}
